Source/Core/DolphinWX/DolphinWX.vcxproj

```diff
@@ -36,10 +36,18 @@
     <Import Project="..\..\VSProps\PCHUse.props" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
+    <IncludePath>$(IncludePath)</IncludePath>
+    <LibraryPath>C:\Users\Nick\Desktop\Games\Emulators\dolphin\Source\Core\DolphinWX\lua5.3;$(LibraryPath)</LibraryPath>
```

Comment:
This doesn't really look portable to me across different systems, can you review and potentially fix this? External libraries should be put in Externals if possible; not inside Source (to make a clear distinction between code we use/include and code that belongs to Dolphin itself)

Reply:
Fixed it to be like all other Dolphin dependencies.